charts/fission-all/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fission-all
-version: 1.5.0
```

Comment:
Please don't change the chart version as part of this PR - that will be changed as part of Fission release 1.6